### DISCUSSION TITLE
Training on a dataset of frequencies

### DISCUSSION THREAD

**Discussion by babyfluffyseal** [openai/whisper#1964] [Q&A]
I would like to be able to train Whisper in brand new "language" datasets that are not related to any languages it has already been trained on. For example, would it be possible to train Whisper to recognize the calling sounds of different birds?

**Comment by phineas-pta** [comment]
u could train on anything u want, but if it's out of whisper original scope, there's no guarantee it returns meaningful result, no one can tell u for sure, u should try and find out
